video-to-frames/video_frames.py

```diff
@@ -0,0 +1,37 @@
+# DOCUMENTATION
```

Comment:
I haven't seen the use case, but are the video and folder names likely to change? They might be better off as parameters passed through the command line if they change a lot.

We'll also work out a good way of getting all dependencies installed as the code base gets built up.

Reply:
Command line arguments would definitely be easier